README.md

```diff
@@ -25,102 +25,119 @@ import valispace
 **And initialize with:**
 
 ```python
-valispace_API = valispace.API()
+valispace = valispace.API()
```

Comment:
Same name for the library and for the API object can get really confusing.

Reply:
Oh I get what you saying. Is just to bad it needs to be so long don't you think? Any ideas? Maybe we can put vali_api?

Reply:
Or we just put vs? vs.get_vali(...)